Examples/Detectors/TGeoDetector/include/ACTFW/TGeoDetector/TGeoDetectorOptions.hpp

```diff
@@ -55,19 +55,19 @@ void addTGeoGeometryOptions(options_t& opt) {
       "Indicator if ring layout is present.")(
       "geo-tgeo-ringtolerance", po::value<read_range>()->default_value({}),
       "Tolerance for ring layout detection in [mm].")(
-      "geo-tgeo-nlayernames", po::value<read_strings>()->default_value({}),
+      "geo-tgeo-nvolume-name", po::value<read_strings>()->default_value({}),
       "Name identifier for negative layer objects, odered along the series.")(
```

Comment:
```suggestion
      "Name identifier for negative volume objects, odered along the series.")(
```

Reply:
I changed to "Name identifier of the volume for searching negative layers."